### PR TITLE
Adding TR language for localization

### PR DESCRIPTION
**Pull request by ishakoktn**

### CODE REVIEW

File: admin_interface/locale/tr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+# Martin Vuelta <martin.vuelta@softbutterfly.io>, 2018
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: django-admin-interface\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2019-09-04 09:18+0000\n"
+"PO-Revision-Date: 2020-01-24 06:07+0300\n"
+"Last-Translator: Martin Vuelta <martin.vuelta@softbutterfly.io>\n"
+"Language-Team: Spanish <dev@softbutterfly.io>\n"
```
Comment (fabiocaccamo) [inline admin_interface/locale/tr/LC_MESSAGES/django.po]: Could you fix `Last-Translator` and `Language-Team` values?

File: admin_interface/locale/tr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+"Language: es\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```
Comment (fabiocaccamo) [inline admin_interface/locale/tr/LC_MESSAGES/django.po]: Since you duplicated an existing translation, is this `Plural-Form` correct for TR localization?

**Comment by fabiocaccamo** [comment]
@ishakoktn any update?

**Comment by ishakoktn** [comment]
I will. Sorry for delay.

File: admin_interface/locale/tr/LC_MESSAGES/django.po (hunk 2 above)
Comment (ishakoktn) [reply admin_interface/locale/tr/LC_MESSAGES/django.po]: It's my fault. I fixed it on new commit.

**Pull request merged**